src/tabnineChatWidget/ChatViewProvider.ts

```diff
@@ -45,6 +45,12 @@ export default class ChatViewProvider implements WebviewViewProvider {
           });
         } catch (e) {
           console.error("failed to handle event. message:", message);
+          void this.chatWebview?.postMessage({
+            id: message.id,
+            payload: {
```

Comment:
I think it will be better to return "error" instead of "payload". wdyt?

Reply:
and this is how we handle it in the app
https://github.com/codota/tabnine-chat-app/compare/yair/handle_not_exists?expand=1